src/main/java/uk/nhs/tis/sync/job/PostFundingSyncJobTemplate.java

```diff
@@ -0,0 +1,30 @@
+package uk.nhs.tis.sync.job;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import javax.persistence.EntityManager;
+import javax.persistence.Query;
+
+/**
+ * This abstract class serves as a template for implementing post funding synchronization jobs
+ * that collects data from a database using native SQL queries.
+ *
+ * @param <T> the type of data objects being synchronized
+ */
+public abstract class PostFundingSyncJobTemplate<T> extends
+    PersonDateChangeCaptureSyncJobTemplate<T> {
```

Comment:
My earlier comments about the class inheritance still apply.  It seems wrong to:
1. create an `abstract` template that only looks like it will have a single concrete class where there are other concrete classes that don't extend a template
2. have a job about Posts extend a template for People

If our model was for animals with a `Dog extends PetTemplate`, and we were adding (non-pet) `Sheep`.  I would be looking for what is common between "Pets" & "Sheep" to build the interface above, maybe in a way that `Sheep extends Livestock`.

Reply:
Let's  have a common super class for both PersonDateChangeCaptureSyncJobTemplate and PostFundingSyncJobTemplate class called CommonDateChangeCaptureSyncJobTemplate. This will probably satisfy the OOP design. What do you think?